src/stackset.ts

```diff
@@ -475,6 +475,11 @@ export interface StackSetProps {
    */
   readonly operationPreferences?: OperationPreferences;
 
+  /**
+   * The input parameters for the stack set template.
+   */
+  readonly parameters?: StackSetParameter;
```

Comment:
`StackSetParameter` should be `StackSetParameters` but i recognize that's not your fault and would be a breaking change :(